global/data/initial-data.json

```diff
@@ -36,50 +23,15 @@
         "1": {
             "id": 1,
             "username": "superadmin",
-            "pronoun": "",
-            "title": "",
-            "first_name": "",
             "last_name": "Administrator",
             "is_active": true,
             "is_physical_person": true,
-            "password": "",
-            "default_password": "",
             "can_change_own_password": true,
-            "gender": null,
-            "email": "",
-            "default_number": "",
-            "default_structure_level": "",
             "default_vote_weight": "1.000000",
-            "last_email_send": null,
-            "is_demo_user": false,
-            "organization_management_level": "superadmin",
-            "is_present_in_meeting_ids": [],
-            "committee_ids": [],
-            "committee_$_management_level": [],
-            "comment_$": [],
-            "number_$": [],
-            "structure_level_$": [],
-            "about_me_$": [],
-            "vote_weight_$": [],
-            "group_$_ids": [],
-            "speaker_$_ids": [],
-            "personal_note_$_ids": [],
-            "supported_motion_$_ids": [],
-            "submitted_motion_$_ids": [],
-            "poll_voted_$_ids": [],
-            "option_$_ids": [],
-            "vote_$_ids": [],
-            "vote_delegated_vote_$_ids": [],
-            "assignment_candidate_$_ids": [],
-            "projection_$_ids": [],
-            "vote_delegated_$_to_id": [],
-            "vote_delegations_$_from_ids": [],
-            "chat_message_$_ids": [],
-            "meeting_ids": []
+	    "meeting_ids": [],
```

Comment:
same for this

Reply:
user/1 meeting_ids is needed.

Reply:
user/meeting_ids is a `NumberArrayField`. These fields are special fields with a default `default` of []. So the checker wants the `user/meeting_ids` field.

Reply:
Updated this.